packages/editor/src/components/editor-canvas/index.js

```diff
@@ -65,270 +67,314 @@ function checkForPostContentAtRootLevel( blocks ) {
 	return false;
 }
 
-export default function EditorCanvas( {
```

Comment:
There's not much changes here, check with "hide whitespace" diff option

Reply:
Why not leave this function as is, then `export default forwardRef( EditorCanvas )` at the end? No diff, and function name is preserved.